src/detections/index.jsx

```diff
@@ -1,4 +1,34 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { Bar } from 'react-chartjs-2'
+import { Grid } from '@material-ui/core'
 
-export const Detections = () => <div>Put components here</div>
+import { labels } from 'detections/labels.json'
+import { ByDays } from 'detections/by-days'
+
+const hours = state => Object.entries(state)
+  .map(([key, value]) => ({
+    label: key,
+    data: value,
+    backgroundColor: `#${((1 << 24) * Math.random() | 0).toString(16)}`,
+  }))
+
+const data = (state, label) => ({
+  labels: label,
+  datasets: hours(state),
+
+})
+
+const useData = (setState, byParameter) => useEffect(() => {
```

Comment:
... why?

Why not use the `byDays` directly, since we import it.

Reply:
done